assignments/this.js

```diff
@@ -13,14 +13,58 @@
 
 // code example for Window Binding
 
+function func(abc) {
+    console.log( this );
+    return abc;
```

Comment:
Your `console.log( this )` is a good example of a Window Binding. Why are you returning `abc` here?